com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs

```diff
@@ -97,8 +122,80 @@ public override void OnInspectorGUI()
             }
 
             EditorGUI.EndDisabledGroup();
+            if (!serializedObject.isEditingMultipleObjects)
+            {
+                DrawStreamSettings(streamPlayer);
+            }
+
+            serializedObject.ApplyModifiedProperties();
+        }
+
+        static void DrawStreamSettings(AlembicStreamPlayer player)
+        {
+            using (var check = new EditorGUI.ChangeCheckScope())
+            {
+                // Draw the stream proeprties
+                var pathSettings = "streamSettings.";
+
+                var so = new SerializedObject(player.StreamDescriptor);
+                so.Update();
+                var settings = so.FindProperty("settings");
+
+                EditorGUILayout.LabelField("Scene", EditorStyles.boldLabel);
+                {
+                    EditorGUI.indentLevel++;
+                    EditorGUILayout.PropertyField(settings.FindPropertyRelative("scaleFactor"),
+                        new GUIContent("Scale Factor",
+                            "How much to scale the models compared to what is in the source file."));
+                    EditorGUILayout.PropertyField(settings.FindPropertyRelative("swapHandedness"),
+                        new GUIContent("Swap Handedness", "Swap X coordinate"));
+                    EditorGUILayout.PropertyField(settings.FindPropertyRelative("interpolateSamples"),
+                        new GUIContent("Interpolate Samples",
+                            "Interpolate transforms and vertices (if topology is constant)."));
+                    EditorGUILayout.Separator();
+
+                    EditorGUILayout.PropertyField(settings.FindPropertyRelative("importVisibility"),
+                        new GUIContent("Import Visibility", "Import visibility animation."));
```

Comment:
I don't get this one...
How would you explain the property in more words?

Reply:
This is btw, a copy paste of the importer UI. I would do an overhaul of both at some point.

Reply:
Basically this options maps alembic visibility to game object active state.
If this option is unticked, GameObject visibility will not be controlled by the alembic file.